scripts/do_shred.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+main() {
+	local -r disk_name='sda'
```

Comment:
As a safety measure, please set `disk_name` to empty. Then exit with error if `disk_name` is empty before running `shred`.

Reply:
Implemented in: e983ee0ec98dca72e5e9ae0f52093abd1041e215